src/Entity/Comments.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\CommentsRepository;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: CommentsRepository::class)]
+#[ORM\HasLifecycleCallbacks]
+class Comments
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
```

Comment:
Why this entity with integer as id ?

Reply:
I have created based on readme 
reference : Create a entity comments with following fields:
id: int
content: text
created_at: datetime
Can you please let me know, If I need to change?

Reply:
Can I add Ulid?

Reply:
Yes it's a better practice, thanks to update it. ;-)

Reply:
Sure, I will do the same. Thank you